todocore/src/main/java/com/knighttodo/todocore/service/privatedb/representation/Day.java

```diff
@@ -31,6 +25,11 @@ public class Day {
     @Column(name = "id")
     private UUID id;
 
+    @CreationTimestamp
+    @Temporal(javax.persistence.TemporalType.DATE)
```

Comment:
you should change it to LocalDate and consider using this annotation after that, just what does it do?

Reply:
Done!